drivers/sensor/everlight/als_pt19/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2025 Dipak Shetty <shetty.dipak@gmx.com>
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_sources_ifdef(CONFIG_EVERLIGHT_ALS_PT19		als_pt19.c)
```

Comment:
You only add this directory if  `CONFIG_EVERLIGHT_ALS_PT19` in `drivers/sensor/everlight/CMakeLists.txt` so this check is redundant. As it's written it looks like you could have an empty (no source) library.